README.md

```diff
@@ -60,29 +73,30 @@ A Makefile has been created to assist with setting up the development environmen
 make setup_dev_environment
 ```
 
+#### Poetry basics
 To install packages within Poetry, use (this will add the dependency to `pyproject.toml`):
 ```bash
 poetry add packagename
 ```
-to install a package for use only in the dev environment, use:
+To install a package for use only in the dev environment, use:
 ```bash
 poetry add packagename --group dev
 ```
 
 ### Running unit tests
 
-In order to run unit tests run the following command:
+In order to run unit tests for the Ingest pipeline run the following command:
 
 ```bash
-make test
+make test_ingest
```

Comment:
As discussed - throughout this change, I believe the testing should be at the root level, not per area i.e. not for ingest?

Reply:
sure, updated